css/css-content/element-replacement-ref.html

```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html>
+<title>Reference for the content CSS attribute can replace an element's contents</title>
+
+<img src='data:image/svg+xml,%3Csvg%20width%3D%22100%22%20height%3D%22100%22%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3Crect%20x%3D%220%22%20y%3D%220%22%20width%3D%22100%22%20height%3D%22100%22%20fill%3D%22green%22%20%2F%3E%3C%2Fsvg%3E' />
```

Comment:
Have you verified that this passes in some browser? I was expected the default style of img vs. p to result in some difference in vertical alignment.

Reply:
It passes in Chrome, but I'm open to improvements if you have any. I couldn't convert it to a JS test (that I could see) because getComputedStyle in Firefox still returned the correct 'content' value, they just don't apply it properly.

Reply:
Thanks, I was just surprised that it would be the same, but the web platform is full of surprises.